renderdoc/api/replay/renderdoc_replay.h

```diff
@@ -1579,6 +1579,10 @@ extern "C" RENDERDOC_API void RENDERDOC_CC RENDERDOC_CheckAndroidPackage(const c
                                                                          const char *exe,
                                                                          AndroidFlags *flags);
 
+DOCUMENT("Internal function that attempts to push Vulkan layer to Android application.");
+extern "C" RENDERDOC_API bool RENDERDOC_CC RENDERDOC_PushLayerToAndroidApp(const char *host,
+                                                                           const char *exe);
```

Comment:
For me as someone unfamiliar with android, the 'push layer' phrasing doesn't immediately suggest that it's installing the layer to the android device to be picked up separately (albeit in an application-specific directory, not globally), rather than inserting it into the apk. Since to me apk == android app.

Is there a more clearly distinct name we could give this function, just to remove any confusion? Maybe mentioning root in some way?